pages/productList/productlist.html

```diff
@@ -0,0 +1,202 @@
+          <div class="sub-section flex-row">
+            <h3 class="heading">Filters</h3>
+            <p>Clear All</p>
+          </div>
+          <hr class="horizontal-line" />
+          <h3 class="heading">Price</h3>
+          <input type="range" class="range" />
+          <hr class="horizontal-line" />
+          <h3 class="heading">Category</h3>
+          <ul class="list">
+            <li>
+              <input type="checkbox" />
+              <span>Sneakers</span>
+            </li>
+            <li>
+              <input type="checkbox" />
+              <span>T-shirt</span>
+            </li>
+            <li>
+              <input type="checkbox" />
+              <span>Perfumes</span>
+            </li>
+          </ul>
+          <hr class="horizontal-line" />
+          <h3 class="heading">Rating</h3>
+          <ul class="list">
+            <li>
+              <input type="radio" name="rating" />
+              <span>4 stars & above</span>
+            </li>
+            <li>
+              <input type="radio" name="rating" />
+              <span>3 stars & above</span>
+            </li>
+            <li>
+              <input type="radio" name="rating" />
+              <span>2 stars & above</span>
+            </li>
+            <li>
+              <input type="radio" name="rating" />
+              <span>1 stars & above</span>
+            </li>
+          </ul>
+          <hr class="horizontal-line" />
+          <h3 class="heading">Sort By</h3>
+          <ul class="list">
+            <li>
+              <input type="radio" />
+              <span>Price-Low to High</span>
+            </li>
+            <li>
+              <input type="radio" />
+              <span>Price-High to Low</span>
+            </li>
+          </ul>
+        </aside>
+
+        <!--Products-->
+        <div class="product-container">
+          <div class="card">
+            <div>
+              <section>
+                <img src="/asset/image/sneaker.jpeg" alt="sneaker" />
+              </section>
+              <section class="sec-2">
+                <h3>Puma</h3>
+                <p class="price-tag"><strong>Rs. 1399</strong></p>
+              </section>
+            </div>
+
+            <section class="sec-3">Vivid one8 Sneakers For Men.</section>
+            <footer>
+              <button class="btn cart-button flex-row">Add to Cart</button>
+            </footer>
+          </div>
+
+          <div class="card">
+            <div>
+              <section>
+                <img src="/asset/image/sneaker.jpeg" alt="sneaker" />
+              </section>
+              <section class="sec-2">
+                <h3>Puma</h3>
+                <p class="price-tag"><strong>Rs. 1399</strong></p>
+              </section>
+            </div>
+
+            <section class="sec-3">Vivid one8 Sneakers For Men.</section>
+            <footer>
+              <button class="btn cart-button flex-row">Add to Cart</button>
+            </footer>
```

Comment:
Reduce button width. It will look good. 
![image](https://user-images.githubusercontent.com/78558235/153807141-1b481708-d1c4-46a4-9309-b98d1f94b644.png)
You can see that it is attached with border.
However, this is optional change.